src/ontology/mondo-edit.obo

```diff
@@ -210715,17 +210715,19 @@ subset: gard_rare {source="GARD:1646"}
 subset: ordo_malformation_syndrome {source="Orphanet:3327"}
 subset: orphanet_rare {source="Orphanet:3327"}
 subset: rare
-synonym: "cutler-Bass-Romshe syndrome" EXACT [Orphanet:3327]
-synonym: "Thyrocerebral-retinal syndrome" RELATED [GARD:0005203]
-synonym: "THYROCEREBRORETINAL syndrome" RELATED [OMIM:274240]
+synonym: "Cutler Bass Romshe syndrome" EXACT [MONDO:0022913]
```

Comment:
curious, why is `Cutler` now capitalized?

Reply:
per the request on the ticket